docs/server/source/simple-network-setup.md

```diff
@@ -0,0 +1,209 @@
+A Node is identified by the triplet `<hostname, node_id, public_key>`.
+
+As a Member, it's your duty to create and store securely your private key, and share your `hostname`, `node_id`, and `public_key` with the other members of the network.
+
+To generate all of that, run:
+
+```
+tendermint init
+```
+
+The `public_key` is stored in the file `.tendermint/config/priv_validator.json`, and it should look like:
+
+```json
+{
+  "address": "5943A9EF6285791A504918E1D117BC7F6A615C98",
+  "pub_key": {
+    "type": "AC26791624DE60",
+    "value": "W3tqeMCU3d4SHDKqrwQWTahTW/wpieIAKZQxUxLv6rI="
+  },
+  "last_height": 0,
+  "last_round": 0,
+  "last_step": 0,
+  "priv_key": {
+    "type": "954568A3288910",
+    "value": "3sv9aExgME6MMjx0JoKVy7KtED8PBiPcyAgsYmVizslbe2p4wJTd3hIcMqqvBBZNqFNb/CmJ4gAplDFTEu/qsg=="
+  }
+}
+```
+
+To extract your `node_id`, run the command:
+
+```
+tendermint show_node_id
+```
+
+Share with all other Members your `node_id`, `pub_key.value`, and the host name or IP of your Node.
+
+**Important note on security: each Member should verify the data they receive from the other Members.**
+
+## Coordinator: Initialize the Network
+At this point the Coordinator should has received the data from all the Members, and should combine them in the `.tendermint/config/genesis.json` file:
+
+```json
+{
+  "genesis_time": "0001-01-01T00:00:00Z",
+  "chain_id": "test-chain-la6HSr",
+  "validators": [
+    {
+      "pub_key": {
+        "type": "AC26791624DE60",
+        "value": "<Member 1 public key"
+      },
+      "power": 10,
+      "name": "<Member 1 name>"
+    },
+    {
+     "pub_key": {
+       "type": "AC26791624DE60",
+       "value": "<Member 2 public key>"
+     },
+     "power": 10,
+      "name": "<Member 2 name>"
+    },
+    {
+     "...": { },
+    },
+    {
+     "pub_key": {
+       "type": "AC26791624DE60",
+       "value": "<Member N public key>"
+     },
+     "power": 10,
+      "name": "<Member N name>"
+    }
+  ],
+  "app_hash": ""
+}
+```
+
+The new `genesis.json` file contains the data that describes the Network.
+
+At this point, the Coordinator must share the new `genesis.json` file with all Members.
+
+## Member: Connect to the other Members
+At this point the Member should have received the `genesis.json` file.
+
+**Important note on security: each Member should verify that the `genesis.json` file contains the correct public keys.**
+
+The Member must copy the `genesis.json` file in the local `.tendermint/config` directory. Every Member now shares the same `chain_id`, `genesis_time`, used to identify the Network, and the same list of `validators`.
+
+The Member must edit the `.tendermint/config/config.toml` file and make the following changes:
+
+```
+...
+
+create_empty_blocks = false
+...
+
+persistent_peers = "<Member 1 node id>@<Member 1 hostname>:46656,\
+<Member 2 node id>@<Member 2 hostname>:46656,\
+<Member N node id>@<Member N hostname>:46656,"
+```
+
+Now the Member must run two commands:
```

Comment:
Something like this?
```
sudo mkdir -p /data/db
sudo chmod -R 700 /data/db
sudo mongod
```

Reply:
This is handled by the operating system when installing MongoDB, in the section *Install MongoDB* I wrote:

> The package manager should take care of installing the startup script for MongoDB as well.

Reply:
Installing MongoDB doesn't also run it, does it?

Reply:
Just to follow-up on and close this comment thread, it seems that our strategy will be to tell people to create startup scripts for MongoDB, BigchainDB and Tendermint, and then to reboot their VM, so that everything will start on reboot. Apparently installing MongoDB already creates a startup script.

Reply:
Yes exactly. I'd like to give hints on how to do things on an Ubuntu machine, and also give some startup scripts. Then it's up to the end user to customize it.